test/system/060-mount.bats

```diff
@@ -173,6 +173,35 @@ load helpers
     run_podman rm -t 0 -f $cid
 }
 
+@test "podman mount containers.conf" {
+    tmpfile1=$PODMAN_TMPDIR/volume-test1
+    random1=$(random_string 30)
+    echo $random1 > $tmpfile1
+
+    tmpfile2=$PODMAN_TMPDIR/volume-test2
+    random2=$(random_string 30)
+    echo $random2 > $tmpfile2
+
+    mountStr1=type=bind,src=$tmpfile1,destination=$tmpfile1,ro
+    mountStr2=type=bind,src=$tmpfile2,destination=$tmpfile1,ro
+    containersconf=$PODMAN_TMPDIR/containers.conf
+    cat >$containersconf <<EOF
+[containers]
+mounts=[ "$mountStr1", ]
+EOF
+
+    run_podman 1 run $IMAGE cat $tmpfile1
+    is "$output" "cat: can't open '$tmpfile1': No such file or directory" "$tmpfile1 does not exist"
```

Comment:
This test looks like a copy-past leftover.

Reply:
No I was proving that the $tmpfile1 does not exist.